pkg/node/node.go

```diff
@@ -107,16 +107,35 @@ func (d Drainer) Drain() error {
 }
 
 // Cordon will add a NoSchedule on the node
-func (d Drainer) cordonNode() error {
-	node, err := d.FetchNode()
+func (n Node) cordonNode() error {
+	node, err := n.FetchNode()
 	if err != nil {
-		log.Println("There was an error which fetching the kubernetes node: ", err)
+		log.Println("There was an error fetching the node in preparation for cordoning: ", err)
 		return err
 	}
-	err = drain.RunCordonOrUncordon(d.drainHelper, node, true)
+	err = drain.RunCordonOrUncordon(n.drainHelper, node, true)
 	if err != nil {
 		log.Printf("Couldn't cordon node %q: %v\n", node, err)
 		return err
 	}
 	return nil
 }
+
+// UncordonNode will remove the NoSchedule on the node
+func (n Node) UncordonNode() error {
+	if n.nthConfig.DryRun {
+		log.Printf("Node %s would have been uncordoned, but dry-run flag was set", n.nthConfig.NodeName)
```

Comment:
Here and below... remember that conversation about just returning a wrapped error to the top-most caller and letting the top-most caller decide whether to print.

Reply:
hmm would you recommend treating the dry-run case as an error with a top-level print?  I was printing here as the actual functionality of `UncordonNode` doing the correct thing when the dry-run flag was enabled.

Reply:
Apologies, not the dry-run ones :) Just the real errors.